simpler-slideshow.py

```diff
@@ -191,7 +191,7 @@ def print_help():
 list_count = []
 list_count_history = []
 num_of_img = 0
-path = "./Photos/"
+path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "Photos", "")
```

Comment:
Some alternatives would be to use: 
- ``Path`` module from [pathlib](https://docs.python.org/3/library/pathlib.html)
	- Path(\_\_file\_\_).absolute().parent
- ``os.get.cwd()``

However, I believe the use of ``os.path`` is sufficient

Reply:
Just realized this fails when turning the script into an executable/application using pyinstaller. If you read the [documentation](https://pyinstaller.org/en/stable/runtime-information.html) of pyinstaller you can recover the application path using ``sys._MEIPASS`` if it is a one-folder bundle or ``sys.executable`` if its a one-file bundle.